### DISCUSSION TITLE
v1.10.2

### DISCUSSION THREAD

**Discussion by jdevcs** [ethereum/web3.js#6383] [Announcements]
### Fixed

- Fixed broken fetch for Node.js > 18.x and fixed double callback (#6381)

<hr /><em>This discussion was created from the release <a href='https://github.com/web3/web3.js/releases/tag/v1.10.2'>v1.10.2</a>.</em>

**Comment by truongnghia** [comment]
Fixed broken fetch for Node.js > 18.x and fixed double callback (Fixed broken fetch for Node.js > 18.x and fixed double callback #6381)
